api/HTMLIFrameElement.json

```diff
@@ -2147,6 +2147,55 @@
           }
         }
       },
+      "setNfcFocus": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/setNfcFocus",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "47",
+              "notes": "Supported in chrome code only."
```

Comment:
I'm not 100% sure, but if this is only supported in chrome code then it isn't exposed to the web, so this should be marked as false.